docs/platforms/python/integrations/celery/index.mdx

```diff
@@ -17,36 +17,89 @@ pip install --upgrade 'sentry-sdk[celery]'
 
 If you have the `celery` package in your dependencies, the Celery integration will be enabled automatically when you initialize the Sentry SDK.
 
-Make sure that the **call to `init` is loaded on worker startup**, and not only in the module where your tasks are defined. Otherwise, the initialization happens too late and events might end up not being reported.
+Make sure that the **call to `sentry_sdk.init()` is loaded on worker startup**, and not only in the module where your tasks are defined. Otherwise, the initialization happens too late and events might end up not being reported.
 
-<PlatformContent includePath="getting-started-config" />
+### Setup Celery (Without Django)
```

Comment:
```suggestion
### Setup Celery Without Django
```

Remove parentheses to be consistent with the "Setup Celery With Django" section. Alternatively, we could rename the "Setup Celery With Django" section to "Setup Celery (With Django)."